src/App.jsx

```diff
@@ -1,13 +1,26 @@
 import React from 'react';
-// import { Routes, Route } from "react-router-dom";
 import Hero from './components/Hero/Hero';
 import './App.css';
+import LatestJobs from './components/LatestJob/LatestJobs';
 
+const data = [
+  { position: 'Web Developer', salary: '2000$-3000$', date: '10/10/2022' },
+  { position: 'Web Developer', salary: '2000$-3000$', date: '10/10/2022' },
+  { position: 'Web Developer', salary: '2000$-3000$', date: '10/10/2022' },
+  { position: 'Web Developer', salary: '2000$-3000$', date: '10/10/2022' },
+  { position: 'Web Developer', salary: '2000$-3000$', date: '10/10/2022' },
+];
```

Comment:
It would be best to put all the mock data into it's own file and import it into here.
You can add a data.js file in src folder and export this data array and the header data and import them here. But if you are just using these data just to test it out then you don't have to add the data for now.

Reply:
btw that data is just for testing purpose if my component working fine or not

Reply:
Oh ok. Then you don't need to add it to the data file.